src/message/html/js.js

```diff
@@ -3,19 +3,19 @@ export default `
 <script>
 function sendMessage(msg) {
   window.postMessage(JSON.stringify(msg), '*');
-}
+};
```

Comment:
Almost certainly these `;` are not needed. (they might be if it was `var a = function`